### ISSUE TITLE
Issue reading reporterConfig.json

### ISSUE THREAD

**Issue by joakim-sch** [YOU54F/cypress-multi-reporters#171]
Wasted a bit of time trying to get this to read my reporter config json file, which kept returning unexpected ' '. Found that adding this trim() fixed the issue.
I switched to using a .js config file, but this might keep someone else from a moments frustration...
#170

**Comment by YOU54F** [comment]
Sorry about this buddy! Is there anything special in your config?

Please note we have moved to another repo

https://github.com/YOU54F/cypress-plugins/tree/master/cypress-multi-reporters

So your pr is best raised against there, will try and get a release out this week for you.

thanks for sharing back

**Comment by joakim-sch** [comment]
Nothing to be sorry about @YOU54F 😄

It was a very basic json, so I have no idea why it found unexpected ' ' (whitespace I presume)

```
{
  "reporterEnabled": "mocha-teamcity-reporter, mochawesome",
  "mochawesomeReporterOptions": {
    "reportDir": "cypress/results",
    "overwrite": false,
    "html": false,
    "json": true
  }
```

Tried creating the file from both Rider and notepad++ without it improving the outcome..

Ill raise the same issue in the new repo location. -> [#1174](https://github.com/YOU54F/cypress-plugins/pull/1174)

**Comment by YOU54F** [comment]
Released now as 1.6.2 from https://github.com/YOU54F/cypress-plugins/actions/runs/3657892446

Will be archiving this repo today, thanks for the change
